### PR TITLE
Support for piecewise split of Job result

### PR DESCRIPTION
**Pull request by mdonaka**
depens on https://github.com/acompany-develop/QuickMPC/pull/4

# Summary
split job result

# Purpose
Successfully save to DB and return to Client when job result is huge.

# Contents
- Change return of lib to MC get_computation_result to stream
- Add ability to combine results in piece order when executing get_computation_result
- Change send_model_param to send split into pieces

# Testing Methods Performed
- pytest
- Send a request to get a join table job with larger data

### CODE REVIEW

File: Src/Utils/ParseCsv.ts [hunk 1]
```diff
@@ -32,8 +32,26 @@ export function parseCsv(data: string): [string[], number[][]]{
...
+    if (pieceSize < lowerLimitSize) {
+        throw new Error(
+          "piece size " + pieceSize.toString() + " is over " +
+          "specified limit size " + lowerLimitSize.toString()
+        );
+    }
+    if (pieceSize > upperLimitSize) {
+        throw new Error(
+          "piece size " + pieceSize.toString() + " is under " +
+          "specified limit size " + upperLimitSize.toString()
```
Comment (come2ry) [inline Src/Utils/ParseCsv.ts:47]: The error message "is under" and "is over" appear to be reversed.

Comment (come2ry) [inline Src/Utils/ParseCsv.ts:47]: @mdonaka 
The error message "is under" and "is over" appear to be reversed.

**Review by come2ry — CHANGES_REQUESTED** [review]

File: Src/Utils/ParseCsv.ts (hunk 1 above)
Comment (mdonaka) [reply Src/Utils/ParseCsv.ts:47]: wow. That's right!
Thanks, fixed it with ade3b719a0250a6ce2897e9c44b8e13c10dca1ef.

File: Src/Client.ts [hunk 2]
```diff
@@ -224,82 +225,94 @@ export class Client {
...
+            });
+
+            promises.push(new Promise((resolve, reject) => {
+                call.on('end', function() {
+                    // concatenate results in pieceId order
+                    resList.sort((a, b):number => a.pieceId - b.pieceId);
+                    let result = "";
+                    let isOk = true;
+                    for (const res of resList) {
+                        result += JSON.parse(res.result);
```
Comment (arukuka) [inline Src/Client.ts:250]: [MEMO] My understanding here is that JSON.parse is used to parse the string.

example:
```
res.result: '"\"[0,"'
JSON.parse(res.result): '"[0.'
```

Comment (arukuka) [reply Src/Client.ts:250]: [MEMO] In TypeScript (JavaScript), `+=` operation is not problem for performance to concat string.

**Review by arukuka — APPROVED** [review]
Thank you for your contribution!  
I checked changes and behavior.

File: Src/Client.ts (hunk 2 above)
Comment (arukuka) [reply Src/Client.ts:250]: I will resolve it.

**Review by come2ry — APPROVED** [review]

**Pull request merged**